src/js/recommendation-ui.js

```
@@ -226,6 +226,10 @@ document.addEventListener('DOMContentLoaded', () => {
          matchedSkillsHtml = `<div class="mt-2 flex flex-wrap gap-1">${skillsList}</div>`;
       }
 
+      const yearsLabel = o.years >= 8 ? 'Veteran' : o.years >= 4 ? 'Experienced' : 'Newcomer';
+      const diffScoreObj = typeof getDifficultyScore === 'function' ? getDifficultyScore(o) : { total: 5 };
```

ESLint

[error] 'getDifficultyScore' is not defined.

(no-undef)